autorest/codegen/models/primitive_schemas.py

```diff
@@ -138,15 +138,14 @@ def imports(self) -> FileImport:
         file_import.add_submodule_import("typing", "Any", ImportType.STDLIB, TypingSection.CONDITIONAL)
         return file_import
 
-class JSONSchema(AnySchema):
+class JSONObjectSchema(AnySchema):
 
     @property
     def docstring_type(self) -> str:
-        return "JSONType"
+        return "JSONObject"
```

Comment:
@changlong-liu we don't want this to be defined as a `JSONObject`, this we still want to be `AnySchema`. You should remove this schema bc we want to just use `Any` in this case